docs/app/Examples/elements/Header/Variations/HeaderVariationsExamples.js

```diff
@@ -0,0 +1,52 @@
+import React, {Component} from 'react';
+import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample';
+import ExampleSection from 'docs/app/Components/ComponentDoc/ExampleSection';
+import {Message} from 'stardust';
+
+export default class HeaderTypesExamples extends Component {
```

Comment:
That would be so, so nice! I feel dumb as all heck because I actually try to go through and check class name /file names before putting in a PR. And yet they still get missed.